phpcs.xml

```diff
@@ -22,6 +22,7 @@
 
 	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
 		<exclude-pattern>tests/*</exclude-pattern>
+		<exclude-pattern>src/*</exclude-pattern>
```

Comment:
In addition to excluding the WordPress sniff for the `src` directory, what about including another sniff that applies _only_ for the `src` directory to enforce PSR-4 filenames?

Reply:
That's a good idea.

Reply:
So far, I couldn't find a sniff for this. But I'll look more.

A sniff that [this issue](https://github.com/squizlabs/PHP_CodeSniffer/issues/2163#issuecomment-422986914) recomments, Slevomat, [requires PHP ^7.2](https://github.com/slevomat/coding-standard/blob/master/composer.json#L12). 

Likewise, [Suin requires >=7.1](https://github.com/suin/phpcs-psr4-sniff/blob/master/composer.json#L30).

Reply:
What about [`Squiz.Sniffs.ClassFileName`](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Standards/Squiz/Sniffs/Classes/ClassFileNameSniff.php)?

Reply:
Ah, good idea. I'll try that out.

Reply:
Great idea, https://github.com/ampproject/amp-wp/pull/3659/commits/951cd3f76b1f67538363a702c0714ed464f47797 adds that rule only for `src/*`